sqlmesh/dbt/loader.py

```diff
@@ -48,9 +49,13 @@ def sqlmesh_config(
     if variables is not None:
         loader_kwargs["variables"] = variables
 
+    connection_kwargs = kwargs.pop("connection_kwargs", {})
```

Comment:
Why do we need this? There are no `connection_kwargs` in the `Config`

Reply:
We don't. Should I do something like this, or is the dict unpacking approach more generally wrong for conditionally passing register_comments if it is not none?

```
    to_sqlmesh_args = {}
    if register_comments is not None:
        to_sqlmesh_args["register_comments"] = register_comments
```